src/Microsoft.AspNetCore.Routing.Abstractions/IEndpointFinder.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public interface IEndpointFinder
+    {
+        IEnumerable<Endpoint> FindEndpoints(Address address);
```

Comment:
This seems to imply and than `Address` has multiple endpoints associated - which probably makes sense.

How do you imagine that addresses are compared for equality?

Reply:
> How do you imagine that addresses are compared for equality?

 Please check this if that's what you mean: https://github.com/aspnet/Routing/pull/546/files#r194552515